silkrpc/commands/rpc_api_table.hpp

```diff
@@ -27,19 +27,22 @@
 #include <nlohmann/json.hpp>
 
 #include <silkrpc/commands/rpc_api.hpp>
+#include <silkrpc/json/stream.hpp>
 
 namespace silkrpc::commands {
 
 class RpcApiTable {
 public:
-    typedef boost::asio::awaitable<void> (RpcApi::*HandleMethod)(const nlohmann::json&, nlohmann::json&);
+    typedef boost::asio::awaitable<void> (RpcApi::*HandleJson)(const nlohmann::json&, nlohmann::json&);
+    typedef boost::asio::awaitable<void> (RpcApi::*HandleStream)(const nlohmann::json&, json::Stream&);
```

Comment:
It was because also HandleStream is an 'HandleMethod'.... The change is to give an idea of what the method is handling. I think it's better to have this naming